src/Connections/GitlabRepo.ts

```diff
@@ -292,6 +296,60 @@ ${data.description}`;
         });
     }
 
+    public async onCommentCreated(event: IGitLabWebhookNoteEvent) {
+        if (this.shouldSkipHook('merge_request.review', 'merge_request.review.comments')) {
+            return;
+        }
+        log.info(`onCommentCreated ${this.roomId} ${this.toString()} ${event.merge_request?.iid} ${event.object_attributes.id}`);
+        const uniqueId = `${event.merge_request?.iid}/${event.object_attributes.author_id}`;
+
+        if (!event.merge_request || event.object_attributes.noteable_type !== "MergeRequest") {
+            // Not a MR comment
+            return;
+        }
+
+        if (event.object_attributes.author_id === event.merge_request.author_id) {
+            // If it's the same author, ignore
+            return;
+        }
+
+        const mergeRequest = event.merge_request;
+
+        const renderFn = () => {
+            const result = this.debounceMRComments.get(uniqueId);
+            if (!result) {
+                // Always defined, but for type checking purposes.
+                return;
+            }
+            const orgRepoName = event.project.path_with_namespace;
+            const comments = result.comments > 1 ? `${result.comments} comments` : '1 comment';
```

Comment:
Can a review have 0 comments?
```suggestion
            const comments = result.comments !== 1 ? `${result.comments} comments` : '1 comment';
```

Reply:
Not really, the function can only be triggered by a comment...so you're going to have at least one.